xtra-libp2p-offer/src/maker.rs

```diff
@@ -45,7 +45,7 @@ impl Actor {
         };
 
         let err_handler =
-            move |e| async move { tracing::debug!(%peer, "Failed to send offers: {e:#}") };
+            move |e| async move { tracing::warn!(%peer, "Failed to send offers: {e:#}") };
```

Comment:
We should totally have these on `warn`. I just wanted to compare the behavior of my `mainnet` taker with the maker logs and just don't have these logs because they are currently on debug and the default log level is info 😅